features/step_definitions/helper_services.rb

```diff
@@ -528,8 +528,10 @@
     logger.info "found existing iSCSI pod, skipping config"
   elsif _pod.exists?(user: admin, quiet: true)
     logger.warn "broken iSCSI pod, will try to recreate keeping other config"
-    @result = admin.cli_exec(:delete, n: _project.name, object_type: "pod", object_name_or_id: _pod.name)
+    @result = admin.cli_exec(:delete, n: _project.name, object_type: "pod", object_name_or_id: _pod.name, force: "true")
```

Comment:
Hi @duanwei33 , it seems the step only use `fore` delete and not wait the iscsi pod accually deleted succeed(disappear).  Is my understanding correct? Maybe still has a bit flake.

Reply:
@Phaow so do you think we need to use without --force or add some additional check for pod existence?

Reply:
Hmm, I think we could use force delete and make sure the fore delete is successful. Just add a wait delete finished(pod disappeared) step after force delete is better.

Reply:
okay, I agree that will be safer

Reply:
@Phaow Updated, PTAL, thanks.